services/wfm/src/main/java/org/openkilda/wfm/topology/flow/FlowTopology.java

```diff
@@ -66,25 +60,22 @@ public StormTopology createTopology() throws NameCollisionException {
         logger.info("Creating FlowTopology - {}", topologyName);
 
         TopologyBuilder builder = new TopologyBuilder();
-        final List<CtrlBoltRef> ctrlTargets = new ArrayList<>();
         Integer parallelism = topologyConfig.getParallelism();
 
         /*
          * Spout receives all Northbound requests.
```

Comment:
Shouldn't the comment be updated along with the renaming NORTHBOUND_KAFKA_SPOUT => FLOW_KAFKA_SPOUT?

Reply:
Renamed.